ai-coding-assistant/mcp-server.md

```diff
@@ -25,70 +27,123 @@ The Telerik Document Processing [MCP (Model Context Protocol) Server](https://mo
+| Command | `"dnx"` | `"dotnet"` | `"npx"` |
+| Arguments | `"Telerik.DPL.MCP", "--yes"` | `"tool", "run", "telerik-dpl-mcp"` | `"-y"` |
+| Server Name | `"telerik-dpl-assistant"` | `"telerik-dpl-assistant"` | `"telerik-dpl-assistant"` |
 
 ### License Configuration
 
-Add your [Telerik license key]({%slug setting-up-license-key%}) as an environment parameter in your `mcp.json` file either as a path or as a script key.
+An active Document Processing license is required to use the Telerik DPL MCP server. Add your [Telerik license key]({%slug setting-up-license-key%}) as an environment parameter in your `mcp.json` file using one of these options:
 
-#### Option 1: License File Path (Recommended)
+* As a license file path (recommended)
 
  ```json
  "env": {
      "TELERIK_LICENSE_PATH": "THE_PATH_TO_YOUR_LICENSE_FILE"
  }
  ```
 
-#### Option 2: Direct License Key
+* As a license key value
 
  ```json
  "env": {
      "TELERIK_LICENSE": "YOUR_LICENSE_KEY_HERE"
  }
  ```
 
-> Option 1 is recommended unless you're sharing settings across different systems. Remember to [update your license key]({%slug setting-up-license-key%}#updating-your-license-key) when necessary.
+> Using a license file path is recommended unless you're sharing settings across different systems. Remember to [update your license key]({%slug setting-up-license-key%}#updating-your-license-key) when necessary.
 
 >note Usually, the `.mcp.json` file is expected to be found in the user's directory: %USERPROFILE%
  
 ## Visual Studio
 
-For complete setup instructions, see [Use MCP servers in Visual Studio](https://learn.microsoft.com/en-us/visualstudio/ide/mcp-servers).
-
-> Early Visual Studio 17.14 versions require the Copilot Chat window to be open when opening a solution for the MCP server to work properly.
+> * Early Visual Studio 17.14 versions require the Copilot Chat window to be open when opening a solution for the MCP server to work properly.
+> * For complete setup instructions, see [Use MCP servers in Visual Studio](https://learn.microsoft.com/en-us/visualstudio/ide/mcp-servers).
 
 ### Workspace-Specific Setup:
 
-1. Add `.mcp.json` to your solution folder:
+1. Add an `.mcp.json` file to either of the following locations:
+
+  * For a workspace-specific setup, add the file to the solution's folder.
+  * For a global setup, add the file to your user directory, `%USERPROFILE%` (for example, `C:\Users\YourName\.mcp.json`).
 
  ```json
+ {
+   "servers": {
+     "telerik-dpl-assistant": {
+       "type": "stdio",
+       "command": "dnx",
+       "args": ["Telerik.DPL.MCP", "--yes"],
+       "env": {
+         "TELERIK_LICENSE_PATH": "THE_PATH_TO_YOUR_LICENSE_FILE",
+         // or
+         "TELERIK_LICENSE": "YOUR_LICENSE_KEY"
+       }
+     }
+   },
+   "inputs": []
+ }
+ ```
+
+  ```json .NET 8/9
+ {
+   "servers": {
+     "telerik-dpl-assistant": {
+       "type": "stdio",
+       "command": "dotnet",
+       "args": ["tool", "run", "telerik-dpl-mcp"],
+       "env": {
```

Comment:
The `env` is not needed in .NET. The license key file is retrieved automatically.